### ISSUE TITLE
builder not accessible in 4.0

### ISSUE THREAD

**Issue by hrj** [scallop/scallop#217]
I have code like this:

```scala
class AbandonCLIConf(arguments: Seq[String]) extends ScallopConf(arguments) {
  override def onError(e: Throwable): Unit = e match {
    case ex: ScallopException => {
      printHelp
      throw e
    }
    case Help("") => {
      builder.printHelp
      throw e
    }
    case Help(subname) => {
      builder.findSubbuilder(subname).get.printHelp
      throw e
    }
    case Version => {
      builder.vers.foreach(println)
      throw e
    }
    case other => super.onError(other)
  }

  ....

}
```

But it fails when updating scallop to version 4.0.0 because `builder` is not accessible. I can see alternate for `builder.printHelp` is now simply `printHelp`. But, can you tell what are the alternatives for:

* `builder.findSubBuilder`
* `builder.vers`

Or more idiomatic ways of achieving the same goal.

Thanks!

**Comment by Rogach** [comment]
That's a regression. I'll make `builder` accessible again.

(it would probably make more sense to include Scallop builder directly in Help or Version objects, to remove all that mess with searching by name - but it will mean more breaking changes and 4.0 version already had enough of them)

**Comment by Rogach** [comment]
Released the fix in version `4.0.1`.
